bin/hermit.hcl

```diff
@@ -0,0 +1,3 @@
+env = {
+  "CGO_ENABLED": "0",
```

Comment:
???

Reply:
We don't use it for prod binaries and, annoyingly, there's a zstd library we use that has an optional CGo build that  takes forever.